docs/source/conf.py

```diff
@@ -26,8 +29,9 @@
 author = 'Edoardo Pasca'
 
 # The short X.Y version
-version = re.sub('^v', '', os.popen('git describe').read().strip())
+version = version.version
 version = version.split('-')[0]
+release = version

```

Comment:
does this retain the mark as nightly build or it will just show the main release tag? It'd be nice to have the commit counter as well.

Reply:
I can remove line 33 and then it will show the full version which includes the commit